### PR TITLE
flickr/find-photos: capture dateTaken, datePublished, dateUpdated

### PR DESCRIPTION
**Pull request by aboodman**

### CODE REVIEW

File: samples/js/flickr/find-photos/src/main.js [hunk 1]
```diff
@@ -71,6 +75,9 @@ const imageType = makeUnionType(sizeTypes.map(st => {
   return makeStructType('', fieldNames, fieldTypes);
 }));
 
+const nsInSecond = Math.pow(10, 9);
+const nsInMillisecond = Math.pow(10, 6);
```
Comment (kalman) [inline samples/js/flickr/find-photos/src/main.js]: Any reason you're not just doing 1e9 and 1e6?

File: samples/js/flickr/find-photos/src/main.js [hunk 2]
```diff
@@ -93,8 +100,20 @@ async function main(): Promise<void> {
         title: v.title,
         tags: new Set(v.tags ? v.tags.split(' ') : []),
         sizes: getSizes(v),
+        datePublished: newStruct('Date', {
```
Comment (kalman) [inline samples/js/flickr/find-photos/src/main.js]: I might pull this into a function like `newDateStruct(Number(v.dateupload) * nsInSecond)` but up to you.

File: samples/js/flickr/find-photos/src/main.js (hunk 1 above)
Comment (arv) [inline samples/js/flickr/find-photos/src/main.js]: Or `10 ** 9` ('m not sure we have es2016 enabled in babel yet)

Comment (kalman) [inline samples/js/flickr/find-photos/src/main.js]: Well this is running on node.

Comment (aboodman) [inline samples/js/flickr/find-photos/src/main.js]: I actually did not know about either of those. Will change.

File: samples/js/flickr/find-photos/src/main.js (hunk 2 above)
Comment (aboodman) [inline samples/js/flickr/find-photos/src/main.js]: Doesn't feel worth it for two, espec when there's a third right there that's different.

**Comment by aboodman** [comment]
PTAL

File: samples/js/flickr/find-photos/src/main.js (hunk 2 above)
Comment (kalman) [inline samples/js/flickr/find-photos/src/main.js]: It's not different? I'm referring to making sure that
- struct is called 'Date'
- property is nsSinceEpoch

```
newDateStruct(Number(v.dateupload) * nsInSecond)
...
newDateStruct(Number(v.lastupdate) * nsInSecond)
...
newDateStruct(Date.parse(v.datetaken) * nsInMillisecond))
```

Comment (aboodman) [inline samples/js/flickr/find-photos/src/main.js]: Oh I see. Yeah I guess. Will do.

Comment (aboodman) [inline samples/js/flickr/find-photos/src/main.js]: Done

**Pull request merged**